workflow-bot-app/src/test/java/com/symphony/bdk/workflow/EngineIntegrationTest.java

```diff
@@ -80,4 +80,32 @@ void deployTwoWorkflowsSameId() throws IOException, ProcessingException {
     Assertions.assertThat(workflowTwo).isExecuted();
   }
 
+  @Test
+  void validateOnlyWorkflow() throws IOException, ProcessingException {
+    final Workflow workflow = SwadlParser.fromYaml(getClass().getResourceAsStream(
+        "/validation/validate-only.swadl.yaml"));
+
+    when(messageService.send(anyString(), any(Message.class))).thenReturn(message("ignored message"));
+
+    engine.parseAndValidate(workflow);
+
+    engine.onEvent(messageReceived("/test"));
```

Comment:
If we test the validation, do we need to trigger an event and use the mock in line 88? We can maybe get ride of lines 88, 92 and 93?